prov/verbs/src/verbs_info.c

```diff
@@ -361,7 +362,12 @@ int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 		return -FI_ENODATA;
 	}
 
-	if (attr->total_buffered_recv > info->rx_attr->total_buffered_recv) {
+	rm_enabled =(info->domain_attr &&
+		     info->domain_attr->resource_mgmt == FI_RM_ENABLED);
+
+	if (!rm_enabled &&
+	    (attr->total_buffered_recv > info->rx_attr->total_buffered_recv))
+	{
```

Comment:
@shefty Is total_buffered_recv a size of a single unexpected message or summarized size of all unexpected messages which a provider is able to process? The note about rendezvous protocol hints at the first case but what if a provider has possibility only for a limited number of unexpected messages, including RTS? Is there some another attribute?